docs/commands/convert.md

```diff
@@ -0,0 +1,12 @@
+# Convert existing oscal documents to another version
+The `convert` command is used to upgrade an existing oscal document to another version. If it fails to upgrade, the errors that require fixing will be outputted to the provided log file (if provided) or `STDERR`
+
```

Comment:
Can you add more information around the high-level "how does this work"? I have hopes that other people will want to come in and contribute - going to make a more concerted effort to have architectural design docs in our projects that describe the logic in plain-english.